Readme.md

```diff
@@ -1,25 +1,23 @@
-<!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
-
-* [Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))
-* [Default.aspx.cs](./CS/WebSite/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/WebSite/Default.aspx.vb))
-<!-- default file list end -->
-# How to bind ASPxGridView created at design mode to different data sources at runtime.
+# Grid View for ASP.NET Web Forms - How to bind the control created in design mode to different data sources at runtime.
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e2965/)**
 <!-- run online end -->
 
+This example demonstrates how to bind [ASPxGridView](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView) with autogenerated columns to different data sources at runtime. The grid is created in design mode.
+
+The project contains [ASPxRadioButtonList](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxRadioButtonList) that allows users to switch between three data sources (SqlDataSource). 
 
-<p>How to bind ASPxGridView with autogenerated columns to different data sources at runtime. The grid is created at design mode.</p><p>This example shows how to select and bind ASPxGridView to a data source at runtime. The page contains ASPxGridView, ASPxRadioButtonList and three data sources (SqlDataSource). Grid's data source is changed by clicking list items. Note  that EnableViewState grid's property is set to false. It is done to avoid exceptions on binding  the grid to another data source.</p><p>Grid's column is autogenerated.</p><p>See also:<br />
-<a href="https://www.devexpress.com/Support/Center/p/E448">How to dynamically switch the grid's data source and recreate columns at runtime</a><br />
-<a href="https://www.devexpress.com/Support/Center/p/E2967">How to bind ASPxGridView with manually created columns to different data sources at runtime. The grid is created in design mode.</a><br />
-<a href="https://www.devexpress.com/Support/Center/p/E2968">How to bind ASPxGridView with autogenerated columns to different data sources at runtime. The grid is created in runtime mode</a></p>
+Note that the grid's [EnableViewState](https://docs.microsoft.com/en-us/dotnet/api/system.web.ui.control.enableviewstate#System_Web_UI_Control_EnableViewState) property is set to `false` to avoid exceptions when binding the grid to another data source.
 
-<br/>
+## Files to Look At
+
+<!-- default file list -->
+- [Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))
+- [Default.aspx.cs](./CS/WebSite/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/WebSite/Default.aspx.vb))
+<!-- default file list end -->
 
+## More Examples
 
+- [Grid View for ASP.NET Web Forms - How to Dynamically Switch the Grid's Data Source and Recreate Columns at Runtime](https://github.com/DevExpress-Examples/aspxgridview-switch-grid-data-source-and-recreate-columns-at-runtime)
```

Comment:
Why does this example title capitalize each word, but the other titles (see lines 1, 22, 23) do not. I think either style is fine, but can we make it consistent?

Reply:
```suggestion
- [Grid View for ASP.NET Web Forms - How to dynamically switch the Grid's data source and recreate columns at runtime](https://github.com/DevExpress-Examples/aspxgridview-switch-grid-data-source-and-recreate-columns-at-runtime)
```